schema/pet-insurance.json

```diff
@@ -148,9 +138,9 @@
             "description": "Available optional extra packages.",
             "typeRef": "#/domains/PetInsurance/types/CoverOptions"
         },
-        "availableVetPaymentTerms": {
+        "availableVetFeeLimits": {
             "description": "Available types of vet bill coverage.",
```

Comment:
This description seems outdated now that the key is "fee limits" instead of "payment terms".